crates/uv-resolver/src/resolver/mod.rs

```diff
@@ -192,12 +191,7 @@ impl<Provider: ResolverProvider, InstalledPackages: InstalledPackagesProvider>
         let requires_python = if markers.is_some() {
             None
         } else {
-            Some(
-                python_requirement
-                    .requires_python()
-                    .map(RequiresPython::to_marker_tree)
-                    .unwrap_or_else(|| MarkerTree::And(vec![])),
-            )
```

Comment:
Nice. You got rid of the `MarkerTree::And(vec![])`! I think the change is for the better right? My understanding is that now it will always have some non-empty marker expression in universal mode. So that when `Requires-Python` isn't provided, it will be based on the installed version of Python? Where as previously, there would be no filtering at all.